docs/_importers/dotclear.md

```diff
@@ -4,3 +4,21 @@ prev_section: csv
 link_source: dotclear
 next_section: drupal6
 ---
+
+## Notes
+
+* Paths are always resolved relative to current directory.
+* Posts will be imported into `_drafts` directory to avoid overwriting any existing file with same filename inside `_posts`.
+* Posts will be imported as HTML files with `.html` file extension.
+* Excerpts if non-empty will be prepended onto content string separated by a blank line i.e. `<excerpt>\n\n<content>`.
+* The same excerpt will be added to front matter to prevent Jekyll from generating a separate excerpt.
+* Existing comments linked to posts in the export-file WILL NOT be imported.
+* "Categories" are not currently imported from the export-file.
+* "Tags" however will be imported and added to relevant posts' front matter.
+* Post URLs are imported from the export-file into front matter with key `original_url`.
+* Jekyll doesn't manage timezone for individual posts. Therefore, timezone metadata of individual posts will be ignored.
```

Comment:
If a timezone offset is present, it would be preferable to preserve it for each post, I think. I always use timezone offsets in my post dates. If it's too hard to extract then we can skip it.

Reply:
I agree with this. Unfortunately, the timezone here is the IANA id (e.g. `Europe/Madrid`) instead of the offset.
In the attached mock export-file, it is `"CET"`.